CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Boring-Stack contributing guidelines
+
+Thank you for taking the time to contribute to our project. We appreciate you! 🫶🏽 Please take a moment to read the following guidelines before contributing:
+
+> ⚠️IMPORTANT **Note**
+>
+> **Pull Requests having no issue associated with them will not be accepted. Firstly get an issue assigned, whether it's already opened or raised by you, and then create a Pull Request.**
+
+## Prerequisites
+Before you contribute to Boring-Stack, make sure you have the following prerequisites:
+
+- Open Source Etiquette: If you've never contributed to an open source project before, have a read of [Basic etiquette](https://developer.mozilla.org/en-US/docs/MDN/Community/Open_source_etiquette) for open source projects.
+
+- Basic familiarity with Git and GitHub: If you are also new to these tools, visit [GitHub for complete beginners](https://developer.mozilla.org/en-US/docs/MDN/Contribute/GitHub_beginners) for a comprehensive introduction to them
+
+- [Node.js](https://nodejs.org/) is installed.
+
+---
+
+## How to Contribute 🤔
+To contribute to Boring-Stack, follow these steps:
+
+- Look at the existing [**Issues**](https://github.com/sailscastshq/boring-stack/issues) or create a new issue if you haven't found a suitable one.
+- [**Fork the Repo**](https://github.com/sailscastshq/boring-stack/issues) and create a branch for the specific issue you are working on. Let's start making a difference together! 😊
+- Create a **[Pull Request](https://github.com/sailscastshq/boring-stack)** (_PR_), which will be reviewed and given suggestions for improvements by the maintatiner.
+- If applicable, include screenshots or screen captures in your Pull Request to help us better understand the impact of your proposed changes.
+
+---
+
+## Submission Guidelines 📝
+
+### Branch Organization
+
+At Boring-Stack, we use the [Gitflow branching model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) where we use a `develop` branch as the main branch for active development. Therefore, all Pull Requests should be directed towards the `develop` branch instead of the `main` branch. This helps us maintain a stable `main` branch while allowing continuous development on the `develop` branch.
+
+### Good First Issue
+
+If you're new to our project, we recommend starting with the [_Good first issue_](https://github.com/sailscastshq/boring-stack/issues) label. These issues are ideal for getting familiar with the project and making your first contribution. Before starting work, please confirm with the maintainers that the issue is still relevant. Feel free to comment on the issue to express your intention and avoid duplicate efforts.
+
+### Sending a Pull Request
+
+To ensure a smooth review process, please follow these guidelines:
+- Fork (https://help.github.com/articles/fork-a-repo) the repository into your own account.
+- In your forked repository, create a new branch: `git checkout -b my-branch main`
```

Comment:
I think this should be `git checkout -b my-branch develop` since we are using gitflow